### PR TITLE
malware domain list feed

### PR DESCRIPTION
**Pull request by dantavori**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added Malware Domain List Feed (need to test)

### CODE REVIEW

File: Packs/FeedMalwareDomainList/Integrations/FeedMalwareDomainList/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## [Unreleased]
+
+
+## [19.11.1] - 2019-11-26
+These notes will be published in the next release notes
```
Comment (avidan-H) [inline Packs/FeedMalwareDomainList/Integrations/FeedMalwareDomainList/CHANGELOG.md]: Remove.

File: Packs/FeedMalwareDomainList/Integrations/FeedMalwareDomainList/FeedMalwareDomainList.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- defaultvalue: indicatorType
```
Comment (avidan-H) [inline Packs/FeedMalwareDomainList/Integrations/FeedMalwareDomainList/FeedMalwareDomainList.yml]:
```suggestion
- defaultvalue: indicatorType
  display: ""
```

File: Packs/FeedMalwareDomainList/Integrations/FeedMalwareDomainList/FeedMalwareDomainList.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  name: feedExpirationPolicy
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+  required: false
+  type: 17
+- defaultvalue: '20160'
+  name: feedExpirationInterval
```
Comment (avidan-H) [inline Packs/FeedMalwareDomainList/Integrations/FeedMalwareDomainList/FeedMalwareDomainList.yml]:
```suggestion
  name: feedExpirationInterval
  display: " "
```

File: Packs/FeedMalwareDomainList/Integrations/FeedMalwareDomainList/FeedMalwareDomainList.yml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    deprecated: false
+    description: Gets the feed indicators.
+    execution: false
+    name: malwaredomainlist-get-indicators
+  isfetch: true
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/python3:3.7.4.2245
```
Comment (avidan-H) [inline Packs/FeedMalwareDomainList/Integrations/FeedMalwareDomainList/FeedMalwareDomainList.yml]:
```suggestion
  dockerimage: demisto/python3:3.7.4.2245
fromversion: 5.5.0
tests:
- Malware Domain List Active IPs Feed Test
```

Comment (avidan-H) [inline Packs/FeedMalwareDomainList/Integrations/FeedMalwareDomainList/FeedMalwareDomainList.yml]: Please update the docker image to the latest one - it's 3.8 something or other.

File: Packs/FeedMalwareDomainList/TestPlaybooks/playbook-Malware_Domain_List_Active_IPs_Feed_test.yml [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+        "height": 735,
+        "width": 380,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs: []
+outputs: []
+sourceplaybookid: AWS Feed Test
```
Comment (avidan-H) [inline Packs/FeedMalwareDomainList/TestPlaybooks/playbook-Malware_Domain_List_Active_IPs_Feed_test.yml]:
```suggestion
sourceplaybookid: AWS Feed Test
fromversion: 5.5.0
```

File: Packs/FeedMalwareDomainList/Integrations/FeedMalwareDomainList/FeedMalwareDomainList.yml [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+      description: The maximum number of results to return. The default value is 50.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: Gets the feed indicators.
+    execution: false
+    name: malwaredomainlist-get-indicators
+  isfetch: true
```
Comment (avidan-H) [inline Packs/FeedMalwareDomainList/Integrations/FeedMalwareDomainList/FeedMalwareDomainList.yml]:
```suggestion
  feed: true
  isfetch: false
```

File: Packs/FeedMalwareDomainList/TestPlaybooks/playbook-Malware_Domain_List_Active_IPs_Feed_test.yml [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
```
Comment (avidan-H) [inline Packs/FeedMalwareDomainList/TestPlaybooks/playbook-Malware_Domain_List_Active_IPs_Feed_test.yml]: This line is incorrect.

**Comment by ronykoz** [comment]
@dantavori merge this on sunday once the build passes, if it will not pass please go to @yuvalbenshalom and figure this thing out together

**Comment by avidan-H** [comment]
Please fix the test playbook asap so we can merge today.

**Comment by avidan-H** [comment]
All the parameter descriptions are standard - no technical writing review required.

**Pull request merged**